packages/oui-datagrid/src/datagrid.controller.js

```diff
@@ -78,6 +78,22 @@ export default class DatagridController {
 
         this.columns = builtColumns.columns;
 
+        this.columns.forEach(column => {
+            if (column.title) {
+                return;
+            }
+
+            column.disableWatcher = this.$scope.$watch(
+                () => this.ouiDatagridColumnBuilder.buildTitle(column.rawTitle, this.getParentScope()),
```

Comment:
Indent could more clear here, declaring those function before using them could be a way to unclutter